tests/test_executor.py

```diff
@@ -0,0 +1,39 @@
+import argparse
+import threading
+import time
+
+from depthai_demo import DepthAI
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-s", "--streams", nargs="+")
```

Comment:
should we set them as `required=True`? If not then `default = None`?

Reply:
`default=None` is added by default by argparse